qt_gui_cpp/src/qt_gui_cpp_sip/CMakeLists.txt

```diff
@@ -31,6 +31,39 @@ set(qt_gui_cpp_sip_INCLUDE_DIRS ${qt_gui_cpp_INCLUDE_DIRS} "${CMAKE_CURRENT_SOUR
 set(qt_gui_cpp_sip_LIBRARY_DIRS ${qt_gui_cpp_LIBRARY_DIRS} lib)
 set(qt_gui_cpp_sip_LDFLAGS_OTHER ${qt_gui_cpp_LDFLAGS_OTHER} -Wl,-rpath,\\"lib\\")
 
+find_library(class_loader_LIBRARY class_loader)
+find_library(console_bridge_LIBRARY console_bridge)
```

Comment:
Neither of these two is directly being used anywhere in this package, no?

Reply:
These are the weird ones. They are brought in through pluginlib_LIBRARIES, but across all platforms they are not provided as absolute paths, and so they are not found during linking. These find_library calls at least choose the correct absolute paths.

Reply:
You simply can't expect this to work. The package doesn't declare a dependency on these and therefore can't expect these packages to be there.

If they are required by `pluginlib` then `pluginlib_LIBRARIES` will already include them. This package doesn't need to / shouldn't know about any recursive dependencies. What of `pluginlib` decides to drop the dependency and do its job differently?

Reply:
I agree with you. I just want to get eyes on this to see why this is even necessary for it to build